From 956c2164cd415fdc0dc0cf5b26498bc6bd457116 Mon Sep 17 00:00:00 2001
From: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
Date: Fri, 22 Sep 2023 09:49:53 +0800
Subject: [PATCH 01/14] PSUSensor: add ina233 support

add device ina233 pmbus reading support

Tested:
Get ina233 reading from PSUSensor by dbus introspect

Change-Id: I4715dac87cbab73038079f9200b3e1cc065b5ea4
Signed-off-by: Delphine CC Chiu <Delphine_CC_Chiu@wiwynn.com>
---
 src/psu/PSUSensorMain.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/psu/PSUSensorMain.cpp b/src/psu/PSUSensorMain.cpp
index 63bc6b5..32ceecd 100644
--- a/src/psu/PSUSensorMain.cpp
+++ b/src/psu/PSUSensorMain.cpp
@@ -86,6 +86,7 @@ static const I2CDeviceTypeMap sensorTypes{
     {"INA230", I2CDeviceType{"ina230", true}},
     {"INA233", I2CDeviceType{"ina233", true}},
     {"INA238", I2CDeviceType{"ina238", true}},
+    {"INA233", I2CDeviceType{"ina233", true}},
     {"IPSPS1", I2CDeviceType{"ipsps1", true}},
     {"IR35221", I2CDeviceType{"ir35221", true}},
     {"IR38060", I2CDeviceType{"ir38060", true}},
-- 
2.47.3

